lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +237,61 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Package(Requirement):
+    def __init__(self, name_string, version_string=None):
+        if version_string is not None:
+            if version_string[0].isdigit():
```

Comment:
This would probably look cleaner:
```python
separator = '==' if version_string[0].isdigit() else ''
name_string = separator.join((name_string, version_string))
```

Reply:
Also, it might make sense to `str.lstrip()` `version_string` before checking the first char.